src/GovUk.Education.ExploreEducationStatistics.Data.Services/ReleaseService.cs

```diff
@@ -113,11 +135,13 @@ private async Task<List<SubjectViewModel>> GetSubjects(Guid releaseId, List<Guid
             return releaseContentBlocks
                 .Where(dataBlock => subjectsToInclude.Contains(dataBlock.Query.SubjectId))
                 .Select(
-                    dataBlock => new FeaturedTableViewModel(
-                        id: dataBlock.Id,
-                        name: dataBlock.HighlightName ?? string.Empty,
-                        description: dataBlock.HighlightDescription ?? string.Empty
-                    )
+                    dataBlock => new FeaturedTableViewModel
+                    {
+                        Id = dataBlock.Id,
+                        Name =  dataBlock.HighlightName ?? string.Empty,
+                        Description = dataBlock.HighlightDescription ?? string.Empty,
+                        SubjectId = dataBlock.Query.SubjectId,
+                    }
```

Comment:
I noticed that this `ReleaseService.ListFeaturedTables(Guid releaseId)` is called by two different controllers, one for listing featured tables by release Id and one by publication Id(which in turn works out the latest published release id first before  calling this.

Do you know if this work will make one of these routes redundant? If so we could mark it for removal so we don't forget to tidy up afterwards.

Reply:
I think this could be covered by the ticket to split featured tables into their own entity, which I'm currently working on, so I will leave a comment for myself to look into this.

Reply:
Commented added here: https://github.com/dfe-analytical-services/explore-education-statistics/blob/b89c646a3208122c551403f7497cb7b1595bbb43/src/GovUk.Education.ExploreEducationStatistics.Data.Api/Controllers/PublicationController.cs